lib/sdk/README.md

```diff
@@ -0,0 +1,52 @@
+# Etcher SDK
+
+## Modules
+
+This section shall give a broad overview of the modules used by the Etcher SDK, and what they're used for.
```

Comment:
I guess this is currently listing just the _external_ modules in the Etcher SDK, and not (yet) including any of the modules in the Etcher repo itself?